.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
-  extends: ['plugin:@typescript-eslint/recommended', 'prettier/@typescript-eslint', 'plugin:prettier/recommended'],
+  extends: ['plugin:@typescript-eslint/recommended', 'prettier', 'plugin:prettier/recommended'],
```

Comment:
```suggestion
  extends: ['plugin:@typescript-eslint/recommended', 'prettier'],
```
`prettier` is enough: https://github.com/prettier/eslint-plugin-prettier